src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +124,18 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sorts all persons in the list.
+     */
+    public void sort() {
+        Collections.sort(internalList, new Comparator<Person>() {
+            @Override
+            public int compare(Person o1, Person o2) {
+                return (o1.getName().toString()).compareTo(o2.getName().toString());
+            }
+        });
+    }
```

Comment:
Nice well commented code. Liked the sleek way you coded your comparator function. Nice job!